Environment/Environment.tex

```diff
@@ -977,7 +981,41 @@ \subsection{The inspector}
 You can change instance variables directly by selecting them, replacing the old value in the right-hand pane by a Squeak expression, and accepting.
 Squeak will evaluate the expression and assign the result to the instance variable.
 
-There are special variants of the inspector for Dictionaries, OrderedCollections, CompiledMethods, and a few other classes that make it easier to examine the contents of these special objects.
+There are many special variants of the inspector for different kinds of collections, methods, \SqVersionSwitch{6.0}{morphs,}{} and a few other classes that make it easier to examine the contents of these special objects.
+\SqVersionSwitch{6.0}{
+For instance, if you \menu{inspect it} an \ct{EllipseMorph} by doing \ct{EllipseMorph new openInWorld; inspect}, you will see a window like that shown in \figref{inspectMorph} that has special fields for accessing the geometry, the screenshot, and the properties of the ellipse.

```

Comment:
I find the "inspect it" somewhat odd, as you are not really referring to the menu action but the general mechanism. Maybe just use "inspect"?

Reply:
```suggestion
For instance, if you inspect an \ct{EllipseMorph} by doing \ct{EllipseMorph new openInWorld; inspect}, you will see a window like that shown in \figref{inspectMorph} that has special fields for accessing the geometry, the screenshot, and the properties of the ellipse.
```